MANIFEST.in

```diff
@@ -8,9 +8,11 @@ include c/enc/*.h
 include c/include/brotli/*.h
 include LICENSE
 include MANIFEST.in
-include python/_brotli.cc
+include python/_brotli.c
```

Comment:
This seems like a typo?
There is no _brotli.cc, just a _brotli.c.
I noticed this in between and was too lazy to make a separate PR for this.